product_docs/docs/biganimal/release/administering_cluster/04_customizing_policy_ definitions.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: "Customizing Azure policy definitions"
+---
+
+Azure Policies help you monitor, identify, and remediate non-compliant resources. Azure automatically assigns a default set of policies to each subscription. If required by your organization, you can further customize these default Azure policies to match BigAnimal's resource configurations. 
+
+!!! Note
+    BigAnimal does not customize your Azure policies to prevent conflicts with external workloads.
+
+### Customize default policy definitions in Azure
+
+Manually perform the following steps in each of your Azure subscriptions:
+
+!!! Note
+    You require *Microsoft.Authorizations/PolicyAssignments/write* permissions to be able to update policy initiatives (sets of policies) in Azure.
+
+
+1. In the Azure portal, enter **Policy** in the search box at the top and open the Policy service.
+
+1. Select **Compliance** on the left side of the Policy page. 
+
+1. On the Compliance page, set the Scope by selecting the ellipsis and then selecting all subscriptions. Click the **Select** button at the bottom of the Scope page to add your selection.
+
+1. You can see a list of all the policy initiatives (sets of policies) automatically assigned by Azure's onboarding process. The policy initiative for each subscription is labeled as *ASC Default (subscription: <Subscription_ID>)*
+
+1. Select a policy initiative in the list and click **Edit assignment**.  
+
+1. On the Edit Initiative Assignment page, click the **Parameters** tab.
+
+1. Clear the **Only show parameters that need input or review** check box.
+
+1. Configure your default ASC policy parameters to only allow BigAnimal's specific configurations. Use the parameter values specified in [Customizable policy definition parameters](#customizable-policy-definition-parameters) to update the parameters.
+
+1. Select the **Review + create** tab at the top of the wizard.
+
+1. Review your selections, then select **Create** at the bottom of the page.
+
+You're now ready to monitor, identify, and remediate non-compliant resources to improve the compliance state of the resources in your subscription.
+
+#### Customizable policy definition parameters
+
+While customizing Azure policies, refer to the following list of recommended parameters and values that are based on BigAnimal's resource configurations. 
+
+Use the values below each parameter while configuring the default ASC policy of a subscription. 
+
+!!! Note
+    
+    JSON values are provided wherever applicable.
+
+
+- **Allowed service ports list in Kubernetes cluster**:
```

Comment:
We should have descriptions accompanying the settings here, I think - we shouldn't ask the customer to do this _completely_ blindly :)

Reply:
Added a little more colour to the recommendations for customising the policy initiative.